docs/DeveloperGuide.md

```diff
@@ -284,66 +284,60 @@ show feedback to user.
 
 
 
-### Filtering Persons
+### Filtering Data
 
 #### Implementation
 
-To allow the user to filter based on any attributes of a `Person`, a `Predicate` class is created
-for each attribute. The `ModelManager` manages all data, which allows us to easily filter `Person`s based
-on their atrributes by passing in the correspionding `Predicate`. 
+The data on the `schedule`, `contacts` and `todos`  tabs can be filtered. The underlying implmentation of the `filter` commnds on each of these tabs are the same, and involves `Predicate`s that looks for entries that match the keyword(s) specified by the user. 
```

Comment:
1. Extra whitespace between "todos" and "tabs"
2. commnds typo
3. Tabs: standardise to Schedule, Contacts and Todos tabs, without enclosing in code blocks

Reply:
changed